src/pages/policy.tsx

```diff
@@ -0,0 +1,56 @@
+import { NextPage } from "next";
+import { Header } from "../components/header";
+import { Layout } from "../components/layout";
+import styles from "../scss/pages/policy.module.scss";
+
+const policyPage: NextPage = () => (
```

Comment:
コンポーネントは PascalCase でお願いします
```suggestion
const PolicyPage: NextPage = () => (
```